htmd/parameterization/ffmolecule.py

```diff
@@ -94,19 +72,37 @@ def __init__(self, filename=None, name=None, rtf=None, prm=None, netcharge=None,
             # If the user has specified explicit RTF and PRM files go ahead and load those
             self._rtf = RTF(rtf)
             self._prm = PRM(prm)
+        elif method == FFTypeMethod.NONE:
+            # Don't assign any atom types
+            pass
         else:
             # Otherwise make atom types using the specified method
-            # (Right now only MATCH)
             fftype = FFType(self, method=self.method)
             self._rtf = fftype._rtf
             self._prm = fftype._prm
-        if not self._rtf or not self._prm:
-            raise ValueError("RTF and PRM not defined")
 
-        self.impropers = np.array(self._rtf.impropers)
+        self.qm = qm if qm else Psi4()
```

Comment:
Sounds cool! Another thing we may want here is to do an `isinstance(qm, QmBase)`, what do you think?

Reply:
Yes, we can add that safeguard!